src/frontend/js/app/AppNavbar/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import UserNavbarItem from './UserNavbarItem';
+
+const NavbarItem = ({ children, to }) => (
+    <li className="nav-item">
+        <Link className="nav-link" to={to}>{children}</Link>
+    </li>
+);
+
+NavbarItem.propTypes = {
+    children: PropTypes.node.isRequired,
+    to: PropTypes.string.isRequired,
+};
+
+const AppNavbar = ({ user, signOut }) => (
+    <nav className="navbar navbar-fixed-top navbar-dark bg-primary">
+        <Link className="navbar-brand" to="/products">New App</Link>
+        <ul className="nav navbar-nav">
+            <NavbarItem to="/products">Home</NavbarItem>
```

Comment:
On otherwise react-router will add it the activé class whatever the current route is. We should an home path